lib/modules/platform/github/massage-markdown-links.ts

```diff
@@ -58,12 +58,19 @@ export function massageMarkdownLinks(content: string): string {
   try {
     const rightSpaces = content.replace(content.trimEnd(), '');
     const matches: UrlMatch[] = [];
-    remark().use(collectLinkPosition(content, matches)).processSync(content);
+    //const newContent = content.replace(regEx(/@&#8203;/g), '@');
+    const newContent = content.replace(
+      regEx(/@&#8203;(?<name>[a-z1-9]*)/g),
```

Comment:
needs to be case insensitive
```suggestion
      regEx(/@&#8203;(?<name>[a-z1-9]*)/gi),
```
or


```suggestion
      regEx(/@&#8203;(?<name>[A-Za-z1-9]*)/g),
```